packages/unsafe-eval/src/install.ts

```diff
@@ -1,39 +1,53 @@
+import { ShaderSystem, unsafeEvalSupported, utils } from '@pixi/core';
 import { syncUniforms } from './syncUniforms';
 
-import type { ShaderSystem, Program, UniformGroup } from '@pixi/core';
+import type { Program, UniformGroup } from '@pixi/core';
 
 interface PIXICore
 {
     ShaderSystem: typeof ShaderSystem;
 }
 
-export function install({ ShaderSystem }: PIXICore): void
+/**
+ * Apply the no `new Function` patch to ShaderSystem in `@pixi/core`.
+ * `@pixi/unsafe-eval` is self-installed since 7.1.0, so this function no longer needs to be called manually.
+ * @param _core
+ * @deprecated since 7.1.0
+ */
+export function install(_core: PIXICore): void
 {
-    if (!ShaderSystem)
-    {
-        throw new Error('Unable to patch ShaderSystem, class not found.');
-    }
+    utils.deprecation('7.1.0', 'install() has been deprecated, @pixi/unsafe-eval is self-installed since 7.1.0');
```

Comment:
One small thing , can you wrap this in `// #if _DEBUG` comment? This makes sure it's excluded from the mini field build.

Reply:
Sure, I will fix this later.

Reply:
Fixed!